spec/unit/puppet_spec.rb

```diff
@@ -24,20 +24,6 @@
     expect(ENV["PATH"]).to eq(newpath)
   end
 
-  it "should change $LOAD_PATH when :libdir changes" do
-    one = tmpdir('load-path-one')
-    two = tmpdir('load-path-two')
-    expect(one).not_to eq(two)
-
-    Puppet[:libdir] = one
-    expect($LOAD_PATH).to include one
-    expect($LOAD_PATH).not_to include two
-
-    Puppet[:libdir] = two
-    expect($LOAD_PATH).not_to include one
-    expect($LOAD_PATH).to include two
-  end
```

Comment:
Previously if the `libdir` setting was changed in puppet.conf, then a daemonized puppet could reload the file and update its `$LOAD_PATH`. However, you would have to also change the `pluginsyncdest` setting so that we download into the same directory that ruby will load from. I'm fairly certain no one does that in practice, and I'd prefer to keep `$LOAD_PATH` immutable once puppet's bootstrapping process is complete. So long story short, I think it's ok to drop that test.